cmd/gateway-gcs.go

```diff
@@ -154,6 +155,19 @@ func gcsToObjectError(err error, params ...string) error {
 	return e
 }
 
+// isValidGCSProjectId - checks if a given project id is valid or not.
+// Project IDs must start with a lowercase letter and can have lowercase
+// ASCII letters, digits or hyphens. Project IDs must be between 6 and 30 characters.
+func isValidGCSProjectID(projectID string) bool {
+	if len(projectID) < 6 || len(projectID) > 30 {
+		return false
+	}
+
+	validProjectID, err := regexp.MatchString("^[a-z][a-z0-9-]*$", projectID)
+	fatalIf(err, "Unable to parse regexp string")
```

Comment:
Don't call fatalIf inside this function.. Keep a global variale for this `regexp.MusCompile()` and then use `projectID.MatchString()`